packages/aila/src/core/Aila.liveWithOpenAI.test.ts

```diff
@@ -13,7 +13,7 @@ const runManually = process.env.RUN_LLM_TESTS === "true";
 
     beforeEach(() => {
       ailaInstance = new Aila({
-        lessonPlan: {},
+        document: { content: {} },
```

Comment:
My thinking here is that we will probably want to pass a schema, a version, some metadata. So the document content probably needs to have this extra level of abstraction.